scheduler.py

```diff
@@ -6,20 +6,24 @@
 from scrape import get_images
 from threading import Thread
 class scheduler():
-    def __init__(self):
+    def __init__(self, conn):
+        directory = os.path.join(CURR_DIR,PICS_FOLDER)
         print (time.time())
-        fetch = Thread(target=self.initFetch)
-        fetch.start()
-        while not os.path.isdir(os.path.join(CURR_DIR,PICS_FOLDER)):
-            print('Downloading images..')
-            time.sleep(60)
+        if conn:
+            fetch = Thread(target=self.initFetch)
+            fetch.start()
+            while not os.path.isdir(os.path.join(CURR_DIR,PICS_FOLDER)):
+                print('Downloading images..')
+                time.sleep(60)
+        elif not os.path.exists(directory):
+                os.makedirs(directory)
         self.change_random()
         self.setStartTime(time.time())
         self.changeCycle()
-        
+
```

Comment:
generally such unrelated changes are made into new commits but i guess this fine for now :)